crates/uplc/src/ast.rs

```diff
@@ -410,6 +410,10 @@ impl Unique {
     pub fn increment(&mut self) {
         self.0 += 1;
     }
+
+    pub fn large_offset(&mut self) {
+        self.0 += 5_000_000;
```

Comment:
🤔 ? Sounds potentially hazardous?

Reply:
The alternative which I like better is when going from NamedDebruijn to Name we also add the unique to the name.

Reply:
This prevents naming collisions causing different scoped lambdas to share the same unique.